NightscoutServiceKit/OTPManager.swift

```diff
@@ -249,9 +249,13 @@ public class OTPManager {
             case .missingOTP:
                 return "Error: Password is required."
             case .expired(let deliveryDate, let maxOTPsToAccept):
```

Comment:
Two versions of the error -- One with the more informative message, showing the age of the notification. The other is a fallback with less info. 

(These ages are useful to help troubleshoot how late iOS is delivering pushes)